modules/rosa-sts-aws-requirements.adoc

```diff
@@ -21,6 +21,100 @@ The following prerequisites must be complete before you deploy a {product-title}
 Customers are encouraged, but not mandated, to deploy resources in a Virtual Private Cloud (VPC) separate from the VPC hosting {product-title} and other Red Hat supported services.
 ====
 
+//
+// Start work for OSDOCS-3198
+//
+[id="rosa-associating-account_{context}"]
+== Associating your AWS account
+
+Performing {product-title} (ROSA) cluster provisioning tasks in {cluster-manager} require linked `ocm-role` and `user-role` resources to your AWS account by using your Amazon Resource Name (ARN).
```

Comment:
@EricPonvelle my understanding is that the OCM role and the user role are AWS IAM resources that are created in a customer's AWS org account. My view is that they are linked to the customer's Red Hat org and user account, respectively.

Specifically:

* The OCM IAM role is linked to the customer's Red Hat organization.
* The user IAM role is linked to the customer's Red Hat user account.

@vkareh and @arendej can you please confirm if my understanding is correct? Thanks!

Reply:
Yes, the linkage here is referring to how the OCM role ARN is stored as a label inside the customer's `Organization` object (in the OCM database) and the user role ARN is stored as a label inside the `Account` object.

Internally we use these to validate that the user is who they say they are (by calling assume role using their Org/Account ID as the AWS External ID), and that OCM has permissions to perform the necessary checks on behalf of the org/user.

The `rosa create ocm-role|user-role` command already does the linking, but there's also the `rosa link ocm-role|user-role` command for this purpose.

Reply:
Thank you for confirming that, Victor!